examples/awx/README.md

```diff
@@ -0,0 +1,25 @@
+# AWX Client Examples
```

Comment:
Consider adding a link to this file from the root README for visibility.

Reply:
Done in the next version of the pull request.